lib/Translation_unit.ml

```diff
@@ -55,7 +55,8 @@ module Error = struct
     Out_channel.write_all n ~data:next ;
     ignore
       (Stdlib.Sys.command
-         (Printf.sprintf "git diff --no-index -u %S %S | sed '1,4d' 1>&2" p n) ) ;
+         (Printf.sprintf "git diff --no-index -u %S %S | sed '1,4d' 1>&2"
+            p n ) ) ;
```

Comment:
I'm not sure I like this change - this was a nice oneliner but anymore after this patch.

Reply:
Just noticing that in this case, we are exceding the margin on this line (81 characters), and it seems it's the best way to split the line (in general the closing `)` are treated in a way to have them get attached to the expression they close).